erp-client/src/app/services/employee/employee.service.ts

```diff
@@ -12,9 +13,13 @@ export class EmployeeService {
 
 	getEmployeeById(id: number): Observable<Employee> {
 		return this.httpClient.get<Employee>(`/api/employees/${id}`);
-	}
-
-	getAllEmployees(): Observable<Employee[]> {
-		return this.httpClient.get<Employee[]>(`/api/employees`);
-	}
+  }
+  
+  getAllEmployees(): Observable<Employee[]> {
+    return this.httpClient.get<Employee[]>('/api/employees').pipe(
+      map((result: any) => {
+        return result._embedded.employees;
```

Comment:
With the added dependency will this exist across the board for get requests? I don't know very much about hal but it looks like it also allows nested _embedded wrappers which may be more challenging: https://stackoverflow.com/questions/47941703/how-to-handle-nested-resources-with-json-hal

I'm cool with trying it out and seeing how it goes though.